wurst/systems/crafting/QuickMakeToolTips.wurst

```diff
@@ -9,44 +9,44 @@ import StringExtensions
 import ToolTipsUtils
 
 // Herbs
-constant let SPECIAL_HERB       = "Special Herb".color(SPECIAL_COLOR)
-constant let BLUE_HERB          = "Blue Herb".color(COLOR_BLUE)
-constant let ORANGE_HERB        = "Orange Herb".color(COLOR_ORANGE)
-constant let PURPLE_HERB        = "Purple Herb".color(COLOR_PURPLE)
-constant let YELLOW_HERB        = "Yellow Herb".color(COLOR_YELLOW)
-constant let RIVER_ROOT         = "River Root".color(COLOR_TEAL)
-constant let RIVER_STEM         = "River Stem".color(COLOR_GRAY)
-constant let ATHELAS_SEED       = "Athelas Seed".color(COLOR_GREEN)
+constant SPECIAL_HERB       = "Special Herb".color(SPECIAL_COLOR)
```

Comment:
We use `let` a lot more often than `constant` and they mean the same thing, so can you keep `let` instead of `constant`?

Reply:
Kinda prefer `constant`over `let` because it seems more explicit to me but okay.

Reply:
Yeah, I'd prefer type modifiers over separate keywords but `let` seems to be preferred throughout Wurst.